src/main/java/com/airlines/dao/impl/AirplaneDaoImpl.java

```diff
@@ -49,15 +52,15 @@ public AirplaneDaoImpl(DataSource dataSource) {
                 return Optional.empty();
             }
         } catch (SQLException e) {
-            throw new DaoOperationException("Error saving product: " + airplane, e);
+            throw new DaoOperationException("Error saving airplane: " + airplane, e);
         }
     }
 
     @Override
     public List<Airplane> findAll() {
         List<Airplane> airplaneArray = new ArrayList<>();
         try (Connection connection = dataSource.getConnection();
-             PreparedStatement preparedStatement = connection.prepareStatement(FIND_ALL_SQL)) {
+             PreparedStatement preparedStatement = connection.prepareStatement(FIND_ALL_QUERY)) {
             ResultSet rs = preparedStatement.executeQuery();
             while (rs.next()) {
                 long id = rs.getLong("id");
```

Comment:
I guess we can remove:
        String codeName = rs.getString("codeName");
                String model = rs.getString("model");
                LocalDate manufactureDate = rs.getDate("manufactureDate").toLocalDate();
                int capacity = rs.getInt("capacity");
                int flightRange = rs.getInt("flightRange");
                Crew crew = (Crew) rs.getObject("crew");

and use it smth like that:

airplaneArray.add(Airplane.builder()
        .withId(rs.getLong("id"))
        .withCodeName(rs.getString("codeName"))
        .withModel(rs.getString("model"))
        .withManufactureDate(rs.getDate("manufactureDate").toLocalDate())
        .withCapacity(rs.getInt("capacity"))
        
        Also we could split declaration and adding:
```
        Airplane airplane = Airplane.builder().withId(rs.getLong....
                                                                .with...
        airplanes.add(airplane);
```